### PR TITLE
#2225 - updates permissions for reviews link

### PR DESCRIPTION
**Pull request by jackkeller**
- updates permissions for Feedback > Reviews ( removes isAdmin, adds hasReportPermissions)

### CODE REVIEW

File: web-ui/src/components/menu/Menu.jsx [hunk 1]
```diff
@@ -71,11 +71,11 @@ const directoryLinks = [
   ['/teams', 'Teams']
 ];
 
-const getFeedbackLinks = (isAdmin, isPDL, isSupervisor) => {
+const getFeedbackLinks = (isAdmin, isPDL, isSupervisor, hasReportPermission) => {
   const links = [];
   if (isAdmin || isPDL) links.push(['/feedback/view', 'View Feedback']);
   links.push(['/feedback/received-requests', 'Received Requests']);
-  if (isSupervisor || isAdmin) links.push(['/feedback/reviews', 'Reviews']);
+  if (isSupervisor || hasReportPermission) links.push(['/feedback/reviews', 'Reviews']);
```
Comment (chrismckiernan) [inline web-ui/src/components/menu/Menu.jsx:78]: is an admin automatically a supervisor so it doesnt need to be in here?

**Review by chrismckiernan — APPROVED** [review]

File: web-ui/src/components/menu/Menu.jsx (hunk 1 above)
Comment (mvolkmann) [inline web-ui/src/components/menu/Menu.jsx:78]: I wonder if this should only check `hasReportPermission`.

Comment (jackkeller) [reply web-ui/src/components/menu/Menu.jsx:78]: Going off of feedback from @mkimberlin for this one.

Comment (jackkeller) [reply web-ui/src/components/menu/Menu.jsx:78]: Unclear, was provided this feedback from @mkimberlin.

File: web-ui/src/components/menu/Menu.jsx [hunk 2]
```diff
@@ -71,11 +71,11 @@ const directoryLinks = [
   ['/teams', 'Teams']
 ];
 
-const getFeedbackLinks = (isAdmin, isPDL, isSupervisor) => {
+const getFeedbackLinks = (isAdmin, isPDL, isSupervisor, hasReportPermission) => {
```
Comment (mkimberlin) [inline web-ui/src/components/menu/Menu.jsx:74]: So, in looking at this again...this isn't actually the right permission set. I'll identify it and drop it to you shortly.

Comment (jackkeller) [reply web-ui/src/components/menu/Menu.jsx:74]: Okay, I'll work in the changes at that point.

Comment (mkimberlin) [reply web-ui/src/components/menu/Menu.jsx:74]: That PR is merged now.

**Comment by jackkeller** [comment]
Aside from `context/selectors.js` and `Menu.jsx` the rest of the files were updated by `yarn format`.

**Review by mvolkmann — APPROVED** [review]

**Pull request merged**